lib/qx/tool/cli/commands/Create.js

```diff
@@ -270,14 +277,19 @@ qx.Class.define("qx.tool.cli.commands.Create", {
               fs.copyFileSync( sourceFile, targetFile );
             }
           } else if (stats.isDirectory()) {
-            if( part === "custom" ) part = values.namespace;
-            let newTargetDir = path.join(targetDir,part);
-            fs.mkdirSync(newTargetDir);
+            let newTargetDir = targetDir;
+            // replace "custon" with namespace, creating namespaced folders in the "class" dir, but not anywhere else
+            let parts = (part === "custom")
+              ? (targetDir.endsWith("class") ? values.namespace.split(/\./) : [values.namespace])
+              : [part];
```

Comment:
if you change this to
let parts = (part === "custom") ? values.namespace.split(/\./) : [values.namespace])  : [part]; 

it will work for resources too

Reply:
Thank you. Can you confirm that the current compiler expects resources in the nested directory structure (see #243)? If yes, I could implement your suggestion and #243 could be closed.

Reply:
all qxl contribs uses nested directory structure for resources.

Reply:
If I do `qx create foo.bar.baz -I && cd foo.bar.baz && qx serve` with the proposed change and the latest qxcompiler, the icon of the default demo application is not found.

Reply:
<img width="1198" alt="bildschirmfoto 2019-01-02 um 13 54 50" src="https://user-images.githubusercontent.com/75390/50592899-5eceb880-0e96-11e9-83bd-e99511ce2663.png">

Reply:
So there is still some issue there. Sorry that I cannot look at it, I have to work on something entirely unrelated currently...

Reply:
That's a Problem of serve!